apps/backoffice-v2/src/common/components/organisms/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { ChevronLeftIcon, ChevronRightIcon } from '@radix-ui/react-icons';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { DayPicker } from 'react-day-picker';
+import { buttonVariants } from '../../atoms/Button/Button';
+
+export type CalendarProps = React.ComponentProps<typeof DayPicker>;
+
+function Calendar({ className, classNames, showOutsideDays = true, ...props }: CalendarProps) {
```

Comment:
Please take consistency into consideration. In a majority of our code this is how we use `export`. Same for any other pasted ShadCN component.
```suggestion
export function Calendar({ className, classNames, showOutsideDays = true, ...props }: CalendarProps) {
```